components/script/dom/bluetooth.rs

```diff
@@ -49,9 +52,77 @@ const NAME_PREFIX_ERROR: &'static str = "'namePrefix', if present, must be nonem
 const NAME_TOO_LONG_ERROR: &'static str = "A device name can't be longer than 248 bytes.";
 const SERVICE_DATA_ERROR: &'static str = "'serviceData', if present, must be non-empty to filter devices.";
 const SERVICE_ERROR: &'static str = "'services', if present, must contain at least one service.";
-const OPTIONS_ERROR: &'static str = "Fields of 'options' conflict with each other.
+pub const OPTIONS_ERROR: &'static str = "Fields of 'options' conflict with each other.
```

Comment:
I think it'd be cleaner like that, but the current solution is also :100:  good. Use the one you prefer.